tests/60app-services/01as-create.pl

```diff
@@ -27,6 +27,40 @@
       });
    };
 
+test "AS can get or create user and return an access_token",
+   requires => [ $main::AS_USER[0], $main::API_CLIENTS[0] ],
+
+   do => sub {
+      my ( $as_user, $http ) = @_;
+
+      do_request_json_for( $as_user,
+         method => "POST",
+         uri    => "/unstable/createUser",
+
+         content => {
+            localpart        => "user_localpart",
+            displayname      => "user_displayname",
+            duration_seconds => 200,
+         },
+      )->then( sub {
+         my ( $body ) = @_;
+
+         assert_json_keys( $body, qw( access_token user_id  home_server ));
+
+         my $user =User( $http , $body->{user_id}, "", $body->{access_token}, undef, undef, undef, [], undef );
+
+         do_request_json_for( $user,
+            method => "GET",
+            uri    => "/r0/profile/:user_id/displayname",
+         )})->then(sub {
```

Comment:
@erikjohnston 
I think here the indentation on line 55 is wrong. Am I wrong so if I change the indentation like below, then it should be fine or?
```
         content => {
            localpart        => "user_localpart",
            displayname      => "user_displayname",
            duration_seconds => 200,
         }
      )->then( sub {
         my ( $body ) = @_;

         assert_json_keys( $body, qw( access_token user_id  home_server ));

         my $user =User( $http , $body->{user_id}, "", $body->{access_token}, undef, undef, undef, [], undef );

         do_request_json_for( $user,
            method => "GET",
            uri    => "/r0/profile/:user_id/displayname",
         )}
      )->then( sub {
         my ( $body ) = @_;

         assert_eq( $body->{displayname}, qw( user_displayname ));

         Future->done(1);
      });
   };
```